src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -353,33 +373,33 @@ private void incomingPacketInfo(
 
             long[] deltas = new long[]{tsDelta, tDelta, sizeDelta};
```

Comment:
is there any reason not to use them by their original names in the code below though? reading `tsDelta` is nicer than `deltas[0]`